baselines/Google Classroom Minimum Viable Secure Configuration Baseline v0.2.md

```diff
@@ -175,9 +175,43 @@ Only teachers SHALL be allowed to unenroll students from classes.
 ### Implementation
 To configure the settings for Student Unenrollment:
 
-#### GWS.CLASSROOM.4.1v0.2 Instructions
+#### GWS.CLASSROOM.5.1v0.2 Instructions
 1.  Sign in to the [Google Admin Console](https://admin.google.com).
 2.  Select **Apps** -\> **Additional Google Service** -\> **Classroom**.
 3.  Select **Student unenrollment**.
 4.  Select **Teachers Only**.
 5.  Select **Save**.
+
+## 5. Class Creation
+
+This section covers who has the ability to create classes.
```

Comment:
I'd recommend adding all the details in the introduction section that the reader will need in order to understand the policy. What is a verified teacher? What is a pending teacher? How does the verification process work? For example:

```suggestion
The first time users sign in to Classroom, they self-identify as either a student or teacher. Users who identify as teachers will be marked as a pending teacher until an administrator verifies them. Google Classroom allows administrators to restrict class creation to only verified teachers.
```